### PR TITLE
Added instructions to run horizontal secure XGBoost in simulator

### PR DESCRIPTION
**Pull request by nvidianz**
### Description

Added instructions on how to copy tenseal context from POC to simulator startup folder.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: docs/user_guide/federated_xgboost/secure_xgboost_user_guide.rst [hunk 1]
```diff
@@ -78,6 +78,8 @@ The following docker image is recommended for GPU training:
 
 Most Linux distributions are supported, as long as they have a recent glibc. The oldest glibc version tested is 2.35. Systems with older glibc may run into issues.
 
+.. _provisioning:
+
 NVFlare Provisioning
```
Comment (YuanTingHsieh) [inline docs/user_guide/federated_xgboost/secure_xgboost_user_guide.rst:83]: we already have this defined in `./docs/programming_guide/provisioning_system.rst:.. _provisioning:`

would this work?

**Review by YuanTingHsieh — COMMENTED** [review]
content is good, should we use the other tag for provision to avoid conflicts with other doc file?

**Review by SYangster — DISMISSED** [review]
LGTM

File: docs/user_guide/federated_xgboost/secure_xgboost_user_guide.rst (hunk 1 above)
Comment (SYangster) [reply docs/user_guide/federated_xgboost/secure_xgboost_user_guide.rst:83]: The local reference would be used before the provisioning_system.rst one

Comment (YuanTingHsieh) [reply docs/user_guide/federated_xgboost/secure_xgboost_user_guide.rst:83]: thanks, good to learn that.

Comment (SYangster) [reply docs/user_guide/federated_xgboost/secure_xgboost_user_guide.rst:83]: Actually I just tested myself to confirm, the local link does work as intended, however now other :ref:`provisioning` in other pages link to this xgboost section. We need rename this tag.

Comment (SYangster) [reply docs/user_guide/federated_xgboost/secure_xgboost_user_guide.rst:83]: @nvidianz

File: docs/user_guide/federated_xgboost/secure_xgboost_user_guide.rst [hunk 2]
```diff
@@ -78,6 +78,8 @@ The following docker image is recommended for GPU training:
 
 Most Linux distributions are supported, as long as they have a recent glibc. The oldest glibc version tested is 2.35. Systems with older glibc may run into issues.
 
+.. _provisioning:
```
Comment (YuanTingHsieh) [inline docs/user_guide/federated_xgboost/secure_xgboost_user_guide.rst:81]: need to rename this

**Review by YuanTingHsieh — CHANGES_REQUESTED** [review]

File: docs/user_guide/federated_xgboost/secure_xgboost_user_guide.rst (hunk 1 above)
Comment (nvidianz) [reply docs/user_guide/federated_xgboost/secure_xgboost_user_guide.rst:83]: Renamed to xgb_provisioning.

File: docs/user_guide/federated_xgboost/secure_xgboost_user_guide.rst (hunk 2 above)
Comment (nvidianz) [reply docs/user_guide/federated_xgboost/secure_xgboost_user_guide.rst:81]: Done.

**Review by YuanTingHsieh — APPROVED** [review]

**Pull request merged**